### PR TITLE
Update test environment warning

### PR DESCRIPTION
**Pull request by mstrasinskis**
# Motivation

To enhance the visibility and effectiveness of the warning message displayed when users are accessing the beta version of the site, we have updated the text and adjusted the button color. These changes are intended to draw more attention to the message, ensuring that users recognize its importance.

# Changes

- Text and style changes.
- Redundant DOM paragraph elements have been removed since their labels are no longer needed.

# Tests

- Tested locally. No changes in the logic.

| Before | After |
|--------|--------|
| image | image | 

# Todos

- [x] Add entry to changelog (if necessary).
Not necessary.

### CODE REVIEW

File: frontend/src/lib/components/warnings/TestEnvironmentWarning.svelte [hunk 1]
```diff
@@ -20,22 +20,18 @@
   disablePointerEvents={true}
 >
   <div class="title" slot="title">
-    <IconWarning />
+    <span class="icon">
```
Comment (peterpeterparker) [inline frontend/src/lib/components/warnings/TestEnvironmentWarning.svelte:23]: According screenshot, the icon within the title is not vertically centered correctly.

**Review by peterpeterparker — APPROVED** [review]
LGTM, thx

File: frontend/src/lib/components/warnings/TestEnvironmentWarning.svelte (hunk 1 above)
Comment (mstrasinskis) [reply frontend/src/lib/components/warnings/TestEnvironmentWarning.svelte]: Good catch, thanks! The screenshot was updated too.

File: frontend/src/lib/components/warnings/TestEnvironmentWarning.svelte [hunk 2]
```diff
@@ -24,18 +24,12 @@
     {$i18n.warning.test_env_title}
   </div>
 
-  <p>{$i18n.warning.test_env_welcome}</p>
```
Comment (aterga) [inline frontend/src/lib/components/warnings/TestEnvironmentWarning.svelte:27]: Could you clarify why this is removed, please?

**Review by aterga — APPROVED** [review]

File: frontend/src/lib/components/warnings/TestEnvironmentWarning.svelte (hunk 2 above)
Comment (mstrasinskis) [reply frontend/src/lib/components/warnings/TestEnvironmentWarning.svelte:27]: There is no separate welcome text anymore, and having a single message is more flexible for editing.

File: frontend/src/lib/components/warnings/TestEnvironmentWarning.svelte [hunk 3]
```diff
@@ -24,18 +24,12 @@
...
-
   <p>
     <Html text={$i18n.warning.test_env_note} />
   </p>
 
-  <p>
-    {$i18n.warning.test_env_request}
```
[the remark this answers is not in the document]
Comment (mstrasinskis) [reply frontend/src/lib/components/warnings/TestEnvironmentWarning.svelte:34]: It’s more flexible for editing to have a single message source. @peterpeterparker, do you recall if there was any important reason for having separate paragraphs?

[the remark this answers is not in the document]
Comment (peterpeterparker) [reply frontend/src/lib/components/warnings/TestEnvironmentWarning.svelte:34]: Cannot recall but, I would guess that it doesn't matter as long as the overall message is understood by the user.

[the remark this answers is not in the document]
Comment (mstrasinskis) [reply frontend/src/lib/components/warnings/TestEnvironmentWarning.svelte:34]: Thank you.

**Pull request merged**